test_aodndata/moorings/test_mooringsProductsHandler.py

```diff
@@ -50,6 +50,10 @@ def test_good_manifest(self):
         upload_broker.upload(INPUT_FILE_COLLECTION)
 
         handler = self.run_handler(GOOD_MANIFEST)
+        self.assertCountEqual(INPUT_FILE_COLLECTION.get_attribute_list('dest_path'),
```

Comment:
the inputs are so long that they occupy the same amount of lines of assigning pointers to the individual variables.

```
e_path = INPUT_FILE_COLLECTION.get_attribute_list('dest_path')
r_path = handler.input_file_collection.get_attribute_list('dest_path')
self.assertEqual(e_path,r_path)
```